packages/cli/src/actions/viewMarket.ts

```diff
@@ -1,16 +1,26 @@
-import SDK from "@zeitgeistpm/sdk";
+// import SDK, { util } from "@zeitgeistpm/sdk";
+import SDK, { util } from "../../../sdk/src";
 
 type Options = {
   endpoint: string;
   marketId: string;
+  seed?: string;
+  address?: string;
 };
 
 const viewMarket = async (opts: Options): Promise<void> => {
-  const { endpoint, marketId } = opts;
+  const { endpoint, marketId, seed } = opts;
+  let { address } = opts;
 
   const sdk = await SDK.initialize(endpoint);
+  
+  if (seed) {
+    address = address || util.signerFromSeed(seed).address
+  }
 
-  const market = await sdk.models.fetchMarketData(Number(marketId));
+  const market = address
+    ? await sdk.models.fetchMarketData(Number(marketId), { address })
```

Comment:
not sure we should tightly bound the market data with a specific user's balance. imo would be better to add a method on the `Market` class to do this instead, `Market.fetchOutcomeBalancesFor(account)`

Reply:
or both?